src/guide/javascript.md

```diff
@@ -191,34 +207,135 @@ const kp = generateKeyPair({
 })
 ```
 
-A basic client setup requires a Torii configuration and an account ID. This
-allows you to perform basic operations like health or status checks. As
-described above, to use transactions or queries you'll need to have a
-`keyPair` parameter as a part of the `Client` instance definition:
+When you have a key pair, you might create a `Signer` using the key pair :
 
 ```ts
-import { Client } from '@iroha2/client'
+import { KeyPair } from '@iroha2/crypto-core'
+import { Signer } from '@iroha2/client'
+import { AccountId, DomainId } from '@iroha2/data-model'
 
-const client = new Client({
-  torii: {
-    // Both URLs are optional in case you only need one of them,
-    // e.g. only the telemetry endpoints
-    apiURL: 'http://127.0.0.1:8080',
-    telemetryURL: 'http://127.0.0.1:8081',
-  },
-  accountId: AccountId({
-    // Account name
-    name: 'alice',
-    // The domain where this account is registered
-    domain_id: DomainId({
-      name: 'wonderland',
-    }),
+// Key pair from previous step
+declare const keyPair: KeyPair
+
+const accountId = AccountId({
+  // Account name
+  name: 'alice',
+  // The domain where this account is registered
+  domain_id: DomainId({
+    name: 'wonderland',
   }),
-  // A key pair, needed for transactions and queries
-  keyPair: kp,
 })
+
+const signer = new Signer(accountId, keyPair)
+```
+
+Well, now we are able to make signatures with `signer.sign(binary)`! But
+usually we need to interact with Iroha and send something to it, like
+transactions or queries. `Torii` will help us.
+
+`Torii` class represents HTTP / WebSocket communications with Iroha. It
+will help us to communicate with each Iroha Endpoint, i.e.:
+
+- Submit transactions
+- Make queries
```

Comment:
not submit?
```suggestion
- Submit queries
```

Reply:
I was always thinking that "submit" is about sending something that implies mutation of a back-end state. In case of transaction "submit" is relevant, but when we do query, we're just reading Iroha state.

Reply:
what about `send queries`?

Reply:
Fine too

Reply:
```suggestion
- Send queries
```